test/steps_test.go

```diff
@@ -2058,3 +2102,457 @@ func tealCheckDryrun(result string) error {
 	}
 	return nil
 }
+
+func createMethodObjectFromSignature(methodSig string) error {
+	abiMethodLocal, err := future.MethodFromSignature(methodSig)
+	abiMethod = abiMethodLocal
+	return err
+}
+
+func serializeMethodObjectIntoJson() error {
+	abiMethodJson, err := json.Marshal(abiMethod)
+	if err != nil {
+		return err
+	}
+
+	abiJsonString = string(abiMethodJson)
+	return nil
+}
+
+func checkSerializedMethodObject(jsonFile, loadedFrom string) error {
+	directory := path.Join("./features/unit/", loadedFrom)
+	jsons, err := loadMockJsons(jsonFile, directory)
+	if err != nil {
+		return err
+	}
+
+	correctJson := string(jsons[0])
+	if correctJson != abiJsonString {
```

Comment:
Note that this type of equality checking for JSON can be unnecessarily strict, since it requires the object field order to be the same.

Instead you could parse both JSON strings into an `interface{}` and then use reflection to check that they are deep equal.

But if it works for now, this can be addressed in a follow up PR when needed.